LuaRules/Gadgets/exp_nuke_effect_chooser.lua

```diff
@@ -42,8 +42,10 @@ function gadget:Explosion_GetWantedWeaponDef()
 end
 
 function gadget:Explosion(weaponID, px, py, pz, ownerID)
-	if (nux[weaponID] and py-GetGroundHeight(px,pz)>100) then
-		return true
+	if (nux[weaponID] and py-GetGroundHeight(px,pz)>200) then
+		Spring.SpawnCEG(defaultInterceptExplosion, px, py, pz, 0, 0, 0, nux[weaponID])
+	else
+		Spring.SpawnCEG(defaultSuccessExplosion, px, py, pz, 0, 0, 0, nux[weaponID])
```

Comment:
IMO leave the success case to be unhandled. This way somebody can tweak the value in the weaponDef and it will take effect even without being aware of this gadget.
```suggestion
		return false
```